src/.nuget/Microsoft.DotNet.CoreCLR.Debug.Development.nuspec

```diff
@@ -39,5 +39,9 @@
     <file src="..\inc\corhdr.h" target="inc\corhdr.h" />
     <file src="..\inc\corinfo.h" target="inc\corinfo.h" />
     <file src="..\inc\corjit.h" target="inc\corjit.h" />
+    <file src="..\inc\opcode.def" target="inc\opcode.def" />
+    <file src="..\inc\openum.h" target="inc\openum.h" />
+    <file src="..\inc\StaticContract.h" target="inc\StaticContract.h" />
+    <file src="..\inc\switches.h" target="inc\switches.h" />
   </files>
```

Comment:
What are the StaticContract.h and switches.h needed for? They should not be needed to describe the CoreCLR JIT interface. (Also, StaticContract.h needs to be in all lowercase to not break Unix build.)